### DISCUSSION TITLE
Is Cypress HIPAA compliant?

### DISCUSSION THREAD

**Discussion by ShyamalaKun** [cypress-io/cypress#28562] [Questions and Help]
We are trying to evaluate Cypress to empower our team with end to end testing tools. Ours is a Healthcare company. Is Cypress HIPAA compliant? If yes, our security team will need Cypress to sign a BAA (Business Associate Agreement ). Will Cypress be willing to sign a BAA?

**Comment by jordanpowell88** [comment]
@ShyamalaKun we would love to connect with you to fully understand your needs. Do you have an email we can contact you at?

**Comment by ShyamalaKun** [comment]
Hi Jordan,

Thank you for the response. My email is ***@***.*** .

Regards,
Shyamala
